src/pages/Index.js

```diff
@@ -1,7 +1,50 @@
 import React from "react"
+import mockApartments from "../mockApartments"
```

Comment:
Data needs to be centralized in App.js so it can eventually come from the database. You only want to import mockApartments once. The apartments in this file should come from props.